Core/Core/Credentials/AccountManager.cs

```diff
@@ -53,8 +54,23 @@ public static async Task<ServerInfo> GetServerInfo(Uri server, CancellationToken
       httpClient
     );
 
+    System.Version version = await gqlClient
+      .GetServerVersion(cancellationToken: cancellationToken)
+      .ConfigureAwait(false);
+    System.Version serverMigrationVersion = new(2, 18, 5);
```

Comment:
These are some magical numbers, I wish there was a better way to specify this. If not, we should add a comment saying why 2.18.5 is the threshold (we know now... but whoever comes fix this in the future may not)